### DISCUSSION TITLE
Hydra Go SDK is wrong?

### DISCUSSION THREAD

**Discussion by Codebreaker101** [ory/hydra#3000] [Q&A] [answered]
I've posted [here](https://github.com/ory/sdk/issues/152) about this issue but haven't heard back yet so I'm posting here.

It seems that the generated `go` files are for Oathkeeper and not for Hydra, from what I can tell. Compared to v1.10.6 the API for v1.11.4 is different and all the file headers say that is for Oathkeeper API.

**Comment by vinckr** [comment, marked as answer]
Hello @Codebreaker101 
That was a quirk in the pipeline and has been resolved: https://github.com/ory/hydra/pull/3014/files,
thanks for pointing it out and apologies for the late response
🏖️
